Source/Core/Core/ConfigManager.h

```diff
@@ -105,6 +105,7 @@ struct SConfig
   bool bEnableCheats = false;
   bool bEnableMemcardSdWriting = true;
   bool bCopyWiiSaveNetplay = true;
+  bool bQoSEnabled = true;
```

Comment:
Please use the new config system (now that it can be used) instead of adding yet another variable to SConfig -- which is already way too huge.

A good place to put this setting would be [NetplaySettings](https://github.com/dolphin-emu/dolphin/blob/master/Source/Core/Core/Config/NetplaySettings.cpp).